modules/ROOT/pages/apikit-workflow-convert-existing.adoc

```diff
@@ -1,104 +1,239 @@
-* Create a main flow, or use an existing one, that includes an HTTP Listener and APIkit Router.
-* Generate backend flows.
-
-The generated backend flows contain the core logic of the API, error handling flows, API Console. You need to add REST resource-action pairing.
-
-== Define the API Definition Files
-
-If you want a RAML-backed API, you can use Studio to create the file: Studio: File > New > RAML API Definition.
-
-image::new-raml.png["Window displays the folder, the name, and the title for creating the new RAML API definition file."]
-
-For example:
-
-. Use the RAML specification and dependent JSON example files provided in the Workflow: Generating a Mule Flow from an API Specification.
-
-. Right-click the name of the RAML in /src/main/resources/api and select Validate. Proceed if there are no problems with the RAML.
-
-. Create any dependent files, or drag and drop them into Studio beside the main RAML. For example, drag the user-example.json to /src/main/resources/api.
+If you chose a backend-first design approach for your Mule project or a Mule application containing business logic, you can convert them into an APIkit project and expose them as a REST service. Fulfill this task following two approaches:
+ 
+* xref:apikit-workflow-convert-existing.adoc#manual-approach[]
+* xref:apikit-workflow-convert-existing.adoc#scaffolding-approach[]
+
+For example, you chose a backend-first implementation approach and you created the following Mule application that retrieves users from a database:
+
+[source,xml,linenums]
+----
+<mule>
+    <http:listener-config name="HTTP_Listener_config">
+		<http:listener-connection host="0.0.0.0" port="8081" />
+	</http:listener-config>
+	<db:config name="Database_Config">
+		<db:my-sql-connection host="localhost" port="3306" user="admin" password="admin" database="usersdb" />
+	</db:config>
+	<flow name="GET-Users-FLOW">
+		<http:listener config-ref="HTTP_Listener_config" path="/users"/>
+		<db:select config-ref="Database_Config">
+			<db:sql ><![CDATA[SELECT * FROM USERS]]></db:sql>
+		</db:select>
+	</flow>
+	<flow name="GET-User-By-Id-FLOW">
+		<http:listener config-ref="HTTP_Listener_config" path="/userById" />
+		<db:select config-ref="Database_Config" >
+			<db:sql ><![CDATA[SELECT * FROM USERS WHERE :userIdParameter == userID]]></db:sql>
+			<db:input-parameters ><![CDATA[#[{userIdParameter: attributes.queryParams.id}]]]></db:input-parameters>
+		</db:select>
+	</flow>
+</mule>
+----
+
+== Manual Approach
+
+You can follow this workflow to configure a functional router for your existing flows manually.
+
+=== Step 1 Design Your API Specification
+
+To define the API specification for your service, see xref:apikit-4-implement-rest-api.adoc#design-your-api-specification[Design Your API Specification at Building, Implementing, and Testing a REST API]. To illustrate this example, create or copy the API specification and JSON example used in that workflow to the `/src/main/resources/api` folder.
```

Comment:
Is this really a topic title? Design Your API Specification at Building, Implementing, and Testing a REST API

Reply:
Since the manual approach and the scaffolding approach have the same 1st step, it may be a good idea to create a "Before You Begin" section requesting the reader to design an API specification (ex step 1). After that, we can describe the 2 approaches: manual and scaffolding. 
I'll rewrite this and come back to you with this option.